covidapi/import_data.py

```diff
@@ -1,6 +1,5 @@
 from db.models import DailyReport
 from db.database import SessionLocal, engine, Base
-from sqlalchemy.orm import Session
```

Comment:
`Session` was imported in two different places but we were using only the one from `requests`